test/metabase/db/schema_migrations_test.clj

```diff
@@ -69,7 +69,7 @@
         (migrate!)
         (doseq [e [e1 e2]]
           (is (= true
-                 (db/exists? User :email (u/lower-case-en e1)))))))))
```

Comment:
this is probably the worst thing I found -- it was supposed to be checking a condition for both values of `e` (`e1` and `e2`) but instead it actually ended up checking `e1` twice and not checking `e2` at all